src/java.base/share/classes/com/sun/crypto/provider/GHASH.java

```diff
@@ -198,6 +199,38 @@ void update(byte[] in, int inOfs, int inLen) {
         processBlocks(in, inOfs, inLen/AES_BLOCK_SIZE, state, subkeyHtbl);
     }
 
+    // Maximum buffer size rotating ByteBuffer->byte[] intrinsic copy
+    static final int MAX_LEN = 1024;
```

Comment:
This filed could be private.  I would like to declare class fields in the beginning of a class, for easy eyes-searching.  Maybe, it is nice to use a multiple of the block size (for example 64 * AES_BLOCK_SIZE), just in case someone else update it to a weird value later.